src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -101,7 +101,9 @@ export const AccessApprovalsTable: React.FunctionComponent<
         accessToken,
       )
       setSearchResult(sortRequest)
-      setArList(sortRequest.results)
+      if (arList) {
+        setArList([...arList, ...sortRequest.results])
+      }
```

Comment:
This is for when sorting after clicking "Load More". Without it would only show sorted ARs from the nextPageToken.